components/FinancialSummary/ExpenseBreakdown.js

```diff
@@ -18,65 +19,77 @@ export default function ExpenseBreakdown() {
                     </div>
                     <div className="grid grid-cols-1 md:grid-cols-3 gap-10 mt-8 mx-3">
 
-                        <div className="bg-white rounded-md shadow-md p-4 flex flex-col items-center">
-                            <div className="text-darkGunMetal">
-                                <div className="flex flex-col items-center">
-                                    <img src="/img/illustrations/MentorshipProgram.webp" alt="Mentorship Program" className="w-1/5 h-auto object-cover rounded-md m-1" />
-                                    <h2 className="text-2xl font-semibold my-2 text-center text-darkGunMetal">Mentorship Program</h2>
+                    <Link href="https://www.asyncapi.com/blog/beyond-boundaries">
+                            <div className="bg-white rounded-md shadow-md p-4 flex flex-col items-center cursor-pointer">
+                                <div className="text-darkGunMetal">
+                                    <div className="flex flex-col items-center">
+                                        <img src="/img/illustrations/MentorshipProgram.webp" alt="Mentorship Program" className="w-1/5 h-auto object-cover rounded-md m-1" />
+                                        <h2 className="text-2xl font-semibold my-2 text-center text-darkGunMetal hover:underline cursor-pointer">Mentorship Program</h2>
```

Comment:
```suggestion
                                        <h2 className="text-2xl font-semibold my-2 text-center text-darkGunMetal">Mentorship Program</h2>
```

Hover properties should be applied to Card, not on heading. You can increase shadow while hovering over the Card. Same for all cards.